src/components/ContactInformation.vue

```diff
@@ -6,11 +6,9 @@
         Lierop, Noord-Brabant <br />
         <br />
         <span class="text__contact-information--bold">Telefoon: </span>
-        <u>06 481 889 92</u> <br />
+          <a href="tel:0648188992">0648188992</a> <br />
```

Comment:
Waarom heb je hier de opmaak van het nummer veranderd?

Reply:
Zodat alle links in lijn is met de rest van de website.

Reply:
Ah, fair. Had het opgebroken omdat dat een stuk makkelijker leest voor dyselecten =p